projects/packages/search/src/instant-search/components/search-result-expanded.scss

```
@@ -55,11 +55,9 @@ $image-square-size: 128px;
 	.jetpack-instant-search__search-result-expanded--no-image & {
 		display: none;
 	}
-}
 
 // Aspect ratio box trick
 // See https://css-tricks.com/aspect-ratio-boxes/
```

If we're going to do this,
```suggestion
	// Aspect ratio box trick
	// See https://css-tricks.com/aspect-ratio-boxes/
```